src/main/java/se/sundsvall/citizenassets/service/mapper/AssetMapper.java

```diff
@@ -0,0 +1,79 @@
+package se.sundsvall.citizenassets.service.mapper;
+
+import static java.util.Collections.emptyList;
+import static java.util.Objects.nonNull;
+import static java.util.Optional.ofNullable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import se.sundsvall.citizenassets.api.model.Asset;
+import se.sundsvall.citizenassets.api.model.AssetCreateRequest;
+import se.sundsvall.citizenassets.api.model.AssetUpdateRequest;
+import se.sundsvall.citizenassets.integration.db.model.AssetEntity;
+
+public class AssetMapper {
```

Comment:
Apart from the rename (which has merit, no doubt) - why did this have to be refactored into a utility class instead of a component?

Reply:
This mapper needed some refactoring as some mapping methods was not working as intended. When the refactoring was done, old habits took over and the mapping methods were created as static methods, making the component-annotation unnecessary.

As existing tests where defined to call real method on the mapperMock, I saw no problem in using static methods in favour to the component annotation.